build_prod.py

```diff
@@ -22,26 +22,26 @@ def copy_file(src, dst, fname):
     dst = os.path.join(dst, fname)
     try:
         os.remove(dst)
-    except OSError, err:
+    except OSError as err:
         if err.errno != os.errno.ENOENT:
```

Comment:
As I found, there is no `os.errno` after python3.7. Only `errno` from module `errno`.
So It's good to do same as for `build_conf.py` - `import errno` and use `errno` without `os.`
Here and three times below.

Reply:
accept

Reply:
Comment is resolved.

Reply:
@Eabramov84 i am not sure it resolves the issue, because, you need to fix version of repo. Besides, some reo of poke may not support python 3 (sumo for example) and this fix may be the source of the issue
anyway, i will check it with some build and put the results here. Than we will come to the decision
hope it works

Reply:
This change needs just to use Python3 for build script not for Yocto

Reply:
we have 2 layers of yocto, and , if low level works with python 2 , we need to switch it during the build.and, it is real problem to build the android , because there is a recursion, and just a way to build it is going into low level yocto and do bitbake android, after that go at high level and continue the build
we need to discuss how to fix it in proper way.

Reply:
I will recheck this issue today. But I'm quite sure that sometime ago I successfully built prod-devel with domA using build script with python3. Quite possible that issue is not in python3 but need to check environment settings. I have both pythons on local host and python2 is default.

Reply:
I can confirm that I successfully built prod-devel domA with build script ported to python3. My build setup is Ubuntu 18 with python2 and python3 installed.
So using python3 for build script doesn't affect upper or lower yocto.